src/components/Card.tsx

```diff
@@ -64,9 +65,16 @@ const appStyles = classnames(
   display('flex'),
   flexDirection('flex-col'),
   alignItems('items-stretch'),
-  minHeight('sm:min-h-card', 'min-h-fit')
+  minHeight('sm:min-h-card', 'min-h-fit'),
+  maxHeight('max-h-app-card'),
+  height('h-app-card'),
+  width('w-full')
```

Comment:
As I can see, there's not that much `mx` in design doc:
![image](https://user-images.githubusercontent.com/49658988/186416756-0b6f46d1-4401-4f78-978a-b9a92de4d31a.png)
But we have pretty big `margin-x`:
![image](https://user-images.githubusercontent.com/49658988/186416795-e633d370-b717-4fff-9dce-77e31605d847.png)